cmd/lncli/commands.go

```diff
@@ -808,7 +811,8 @@ var closeChannelCommand = cli.Command{
 			Name: "sat_per_vbyte",
 			Usage: "(optional) a manual fee expressed in " +
 				"sat/vbyte that should be used when crafting " +
-				"the transaction",
+				"the transaction. Default is a conf-target of" +
```

Comment:
nit: missing space between "of" and "6 blocks". Also, since these aren't full sentences in the first place, should we just use semicolons? e.g. "the transaction; default is a conf-target ..."